bucoffea/config/vbfhinv.yaml

```diff
@@ -221,8 +221,6 @@ default:
       use_average: False
 
   run:
-    sync: False
-    ulegacyv8: True
```

Comment:
Thanks for removing the `ulegacyv8` variable, I think it will simplify things :)